### DISCUSSION TITLE
atomci  1.6.x 我想做什么/想要什么

### DISCUSSION THREAD

**Discussion by colynn** [go-atomci/atomci#124] [General]
`atomci 1.5.0` 马上release，做好很多很赞的调整（应用定义全局唯一，数据迁移，单元测试，还有一些集成项配置的统一与简化， 但是对于应用全流程的cicd还是缺少些东西，所以从你的角度，你想下一版本做什么或是有什么期待，欢迎讨论/留言

**Comment by fanhousanbu** [comment]
want:
1. a health unit tests coverage，what and how to be a good PR
2. a vision: WHY 1.6, NOT 1.9?
3. a good look LOGO 💯 

want to do:
1. divided CI & CD
2. WYSIWYG yaml editor
3. bring more stars

**Comment by sampsonye** [comment]
1.6.x 我的理解还是基于**功能完善度**的，因此我期望：
1. 发布记录和发布**回滚**功能
2. 用户管理能力增强，有“项目团队”概念
3. 支持集成登录

**Reply by colynn** [reply]
可以具体阐述下**回滚**的具体的需求场景吗，是不是可以通过再次的正向发布来解决。

**Reply by sampsonye** [reply]
以应用为核心的发布时间轴，是应用能够回滚的根本所在

**Comment by gosszhang** [comment]
功能：增加制品仓库，汇总所有CI产物。
优化：建议升级Beego到最新版本，使用注解路由代替手动配置路由。

**Comment by sampsonye** [comment]
另 #131  我们需要考虑的Jenkins安全风险
1. 移除对Jenkins的依赖，重新设计构建历史详情（成本大，改动多，效果好，符合长期发展）
2. 调用Jenkins相关API获取构建详情内容，由Atomci自行渲染（成本大，改动小，但不符合长期发展）
3. 通过atomci转发对Jenkins的请求（成本小，改动小，适合短期发展）

**Reply by colynn** [reply]
先执行3， 再执行1， 第2点我们就可以放弃了

**Comment by colynn** [comment]
feature:
- divided CI & CD
- 增加制品仓库
- 基本的用户管理功能（权限功能正常工作）

improvement:
-  降低jenkins的接入成本，提高安全性（屏蔽jenkins直接通过前端访问）
- 核心功能单元测试覆盖率提高

**Reply by colynn** [reply]
improvement: 
项目的代码目录结构不太清晰

**Comment by colynn** [comment]
会议主题：atomci 1.x 版本讨论
会议时间：2022/05/16 20:15-21:00 (GMT+08:00) 中国标准时间 - 北京

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/dm/sjhezFVEM8Py

#腾讯会议：682-935-718
